docs/rules/built-in-rules.md

```diff
@@ -91,43 +104,41 @@ Details of all the rules available "out of the box" with Redocly CLI are listed
 - [tags-alphabetical](./oas/tags-alphabetical.md): Tags in the top-level `tags` section must appear alphabetically
 
-## Rule configuration syntax
+## AsyncAPI rules
+
+Use the rules in this section for AsyncAPI-specific linting.
+Other rules such as the `spec` and `info.*` rules also apply to AsyncAPI.
 
-To change your settings for any given rule, add or modify its corresponding entry in your Redocly configuration file.
+- [channels-kebab-case](./async/channels-kebab-case.md): Channels must be in `kebab-case` format
+- [no-channel-trailing-slash](./async/no-channel-trailing-slash.md): No trailing slashes on channels
 
-You can specify global settings in the top-level `rules` object, or use per-API settings by adding a `rules` object under each API in `apis`.
+## Arazzo rules
 
-You can format each entry in the `rules` object in one of the following ways:
+Within the Arazzo family of rules, there are rules for the main Arazzo specification format, and some additional rules for extensions supported by Spot, the Redocly testing utility.
 
-- Short syntax with single-line configuration `rule-name: {severity}`, where `{severity}` is one of `error`, `warn` or `off`. You can't configure additional rule options with this syntax.
+### Arazzo
 
-```yaml
-apis:
-  main:
-    root: ./openapi/openapi.yaml
-    rules:
-      specific-api-rule: warn
-rules:
-  example-rule-name: error
-```
+- [criteria-unique](./arazzo/criteria-unique.md): the criteria list must not contain duplicated assertions
+- [parameters-unique](./arazzo/parameters-unique.md): the `parameters` list must not include duplicate parameters
+- [requestBody-replacements-unique](./arazzo/requestBody-replacements-unique.md): the `replacements` of the `requestBody` object must be unique
+- [sourceDescriptions-name-unique](./arazzo/sourceDescriptions-name-unique.md): the `name` property of the `sourceDescription` object must be unique across all source descriptions
+- [sourceDescriptions-type](./arazzo/sourceDescriptions-type.md): the `type` property of the `sourceDescription` object must be either `openapi` or `arazzo`
+- [stepId-unique](./arazzo/stepId-unique.md): the `stepId` must be unique amongst all steps described in the workflow
+- [step-onFailure-unique](./arazzo/step-onFailure-unique.md): the `onFailure` actions of the `step` object must be unique
+- [step-onSuccess-unique](./arazzo/step-onSuccess-unique.md): the `onSuccess` actions of the `step` object must be unique
+- [workflow-dependsOn](./arazzo/workflow-dependsOn.md): the items in the `workflow` `dependsOn` property must exist and be unique
+- [workflowId-unique](./arazzo/workflowId-unique.md): the `workflowId` property must be unique across all workflows
+- [sourceDescriptions-not-empty](./arazzo/sourceDescriptions-not-empty.md): the `sourceDescriptions` must be defined and the list must have at least one entry.
 
-- Verbose syntax, where you can configure additional options for rules that support them.
+### Spot [ deprecated ]
```

Comment:
What is deprecated in this section, the rules? We need to add more context, because the section title might be understood as "Spot is deprecated".